nixos/modules/services/security/vaultwarden/default.nix

```diff
@@ -217,6 +277,27 @@ in {
         User = mkDefault user;
         Group = mkDefault group;
         ExecStart = "${pkgs.bash}/bin/bash ${./backup.sh}";
+        ReadOnlyDirectories = mkIf (!hasPrefix "/var/lib/bitwarden_rs" cfg.backupDir) [ "/var/lib/bitwarden_rs" ];
```

Comment:
As you wrote earlier, we can't easily access the default value of DATA_FOLDER. Can we maybe use configEnv.DATA_FOLDER?

And also update systemd.services.backup-vaultwarden to use that?

Reply:
Going to test this locally, assuming we also want to move to `/var/lib/vaultwarden` as the default? though wont that break current uses

Reply:
Alright seems to work for my use case ( after moving /var/lib/bitwarden_rs to /var/lib/vaultwarden of course ) but would appreciate some testing of the the backup service from someone who had already been using it.

Reply:
> Going to test this but not super sure I follow sorry, assuming we also want to move to `/var/lib/vaultwarden` as the default?

Eventually yes. 

> though wont that break current uses?

Yeah, it will be a breaking change.



> also don't understand the logic behind squashing this change into the additional hardening one as they seem like fairly separate concerns with varying influences on potential issues

That should be a separate commit, yes. I only want to have it ready when we switch the default.